geode-core/src/main/java/org/apache/geode/distributed/internal/LocatorLoadSnapshot.java

```diff
@@ -85,49 +86,52 @@ public LocatorLoadSnapshot() {
   }
 
   public void addServer(ServerLocation location, String[] groups, ServerLoad initialLoad) {
-    addServer(location, groups, initialLoad, 30000);
+    addServer(location, "", groups, initialLoad, 30000);
```

Comment:
It looks like this is only used by tests, is that right? It might be better to remove this method, which uses and incorrect memberId of ""

Reply:
I have just added a commit to fix this.